static/src/javascripts/lib/raven.js

```diff
@@ -57,12 +58,18 @@ const sentryOptions = {
         const isIgnored =
             typeof data.tags.ignored !== 'undefined' && data.tags.ignored;
         const { enableSentryReporting } = config.get('switches');
-        const isInSample = Math.random() < 0.008; // 0.8%
+        const isSentinelLoggingEvent = data.tags.tag === 'commercial-sentinel'
```

Comment:
Wondering if its possible for `data.tags` to be `undefined` ? If so might be good to be defensive here with some optional chaining ie `data?.tags?.tag`

Reply:
You're right, `lib/raven` is used by other modules, not just `sentinel.ts`, so I think it's definitely possible for `data.tags` to be undefined.